doc/cli_guidelines.md

```diff
@@ -192,6 +192,10 @@ There are pending questions. Please, answer questions first: agama questions.
 
 ## Non Interactive Mode
 
-Commands should offer a `--non-interactive` option to make scripting possible. The non interactive mode should allow answering questions automatically.
+There are questions subcommand that support `mode` subcommand to set non-interactive mode. Another option is to use `answers` subcommand to pass file with
```

Comment:
NP: I woud rephase it as: There is a `questions` subcommand that support `mode` and `answers` subcommands. The `mode` subcommand allows to set `interactive` and `no-interactive` modes. And `answers` allows to pass a file with prefefined answers.